src/SpotifyWebAPI.php

```diff
@@ -364,15 +365,15 @@ public function changeVolume($options)
      *
      * @return array|object The new playlist. Type is controlled by the `return_assoc` option.
      */
-    public function createPlaylist($options)
+    public function createPlaylist($userId, $options)
     {
         $options = json_encode($options);
 
         $headers = [
             'Content-Type' => 'application/json',
         ];
 
-        $uri = '/v1/me/playlists';
+        $uri = '/v1/' . $userId . '/playlists';
```

Comment:
We usually wanna support passing a Spotify URI too, there's a helper method to extract the ID from it:

```suggestion
        $userId = $this->uriToId($userId, 'user');
        $uri = '/v1/' . $userId . '/playlists';
```